modules/gsp-cluster/service-operator.tf

```diff
@@ -127,3 +128,21 @@ resource "aws_iam_policy" "service-operator-managed-role-permissions-boundary" {
   description = "Permissions boundary for roles created by the service operator"
   policy      = "${data.aws_iam_policy_document.service-operator-managed-role-permissions-boundary.json}"
 }
+
+resource "aws_security_group" "rds-from-worker" {
```

Comment:
I wonder if it would be better to have the operator set this up...As we add more capabilities to the operator it would be better to not have to add terraform to the GSP in order to support them. Not something for this PR/card but certainly something we should consider, I think.

Reply:
They all have to go into the GSP repository so I'm not sure if it makes sense to have to add lots of extra code for these static things when we could just terraform them. I guess maybe we wouldn't have to pipe around as much stuff...